lib/jxl/encode.cc

```diff
@@ -1437,6 +1438,11 @@ JxlEncoderStatus JxlEncoderAddJPEGFrame(
                            "Error setting basic info");
     }
   }
+  if (!io.blobs.exif.empty()) {
+    JxlOrientation orientation;
+    jxl::InterpretExif(io.blobs.exif, &orientation);
```

Comment:
This fails the tests, you should pass &frame_settings->enc->metadata.m.orientation directly

Reply:
I had tried that before, but `frame_settings->enc->metadata.m.orientation` is uint32_t and InterpretExif expects JxlOrientation...
Initializing should fix the test, hopefully.